e2e/pages/widgets/shared/select.js

```diff
@@ -0,0 +1,36 @@
+import { popover } from "e2e/support/helpers";
+
+export class Select {
+  static byTestId = testId => {
+    return new Select(() => cy.findByTestId(testId));
+  };
+
+  static byLabel = label => {
+    return new Select(() => cy.findByLabelText(label));
+  };
+
+  constructor(locator) {
+    this.locator = locator;
+  }
+
+  select = value => {
+    this.locator().click();
+    popover().within(() => {
```

Comment:
Can we add a version of `Popover` in `shared` that wraps this helper? I'd like us to avoid using old helpers directly if possible